@ndlib/gatsby-theme-marble/src/components/Common/style.module.css

```diff
@@ -0,0 +1,90 @@
+
+.inputBox {
+  width: 95%;
+  background-color: khaki;
+  border: 1px solid grey;
+  height: 150px;
+  display:block;
+  padding: 10px;
+  margin-top: 0px;
+  margin-left: auto;
+  margin-right: auto;
+}
+.inputBoxLabel {
+  width: 95%;
+  margin-bottom: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 20px;
+  float: left;
+}
+.inputText {
+  width: 95%;
+  background-color: khaki;
+  height: 30px;
+  display:block;
+  padding-left: 10px;
+  margin-top: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  border: 1px solid grey;
+}
+.inputTextLabel {
+  width: 95%;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 10px;
+  display: block;
+}
+.formText {
+  color: red;
+  font-size: .8em;
+  padding-left: 20px;
+}
+.inputRadio {
+  float: left;
+  margin-top: 12px;
+}
+.inputRadioLabel {
+  float: left;
+  margin-top: 10px;
+  padding-left: 15px;
+}
+.alert {
+  color: red;
+  font-size: .8em;
+  padding-top: 5px;
+  padding-left: 20px;
+}
+.submit {
+  width: 90px;
+  height: 35px;
+  background-color: #007d89;
+  color: #FFFFFF;
+  font-size: 18px;
+  margin: 12px;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+.button {
+  width: 90px;
+  height: 35px;
+  background-color: #007d89;
+  color: #FFFFFF;
+  font-size: 18px;
+  margin: 12px;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+.button:disabled {
+  width: 90px;
+  height: 35px;
+  background-color: #c2dee0;
+  color: #FFFFFF;
+  font-size: 18px;
+  margin: 12px;
+  text-transform: uppercase;
+  cursor: pointer;
+}
```

Comment:
This css file could be significantly condensed. For instance, The `.button` and `.submit` classes are identical, so they don't need two separate rules. Also bear in mind that pseudo-classes (like `:disabled`) still inherit the styles from the class, so there is no reason to repeat the same rules for width, color, margin, etc. Only list the things that are different while disabled (in this case, just `background-color`)

Some of the other styles in this file could also be simplified in a similar manner by putting the common input styles in one rule and having separate rules just for the differences between box, label, etc.